sumologic/resource_sumologic_monitors_library_monitor_test.go

```diff
@@ -523,7 +531,44 @@ resource "sumologic_monitor" "test" {
 	  }
 	playbook = "This is a test playbook"  
 	alert_name =  "Alert from {{Name}}"
-}`, testName)
+	obj_permission {
+      subject_type = "role"
+      subject_id = sumologic_role.tf_test_role_01.id
+      permissions = ["Read","Update","Delete"]
+    }
+    obj_permission {
+      subject_type = "role"
+      subject_id = sumologic_role.tf_test_role_02.id
+      permissions = ["Read"]
+    }
```

Comment:
nit: more proper indent for `obj_permission` ? 
From: 
```
	alert_name =  "Alert from {{Name}}"
	obj_permission {
      subject_type = "role"
      subject_id = sumologic_role.tf_test_role_01.id
      permissions = ["Read","Update","Delete"]
    }
    obj_permission {
      subject_type = "role"
      subject_id = sumologic_role.tf_test_role_02.id
      permissions = ["Read"]
    }
```
to :
```
	alert_name =  "Alert from {{Name}}"
	obj_permission {
              subject_type = "role"
              subject_id = sumologic_role.tf_test_role_01.id
              permissions = ["Read","Update","Delete"]
        }
        obj_permission {
              subject_type = "role"
              subject_id = sumologic_role.tf_test_role_02.id
              permissions = ["Read"]
        }
```

Reply:
I used make fmt to fix indentation issues.